addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml

```diff
@@ -7,7 +7,7 @@
                 <t t-if="threadViewTopBar.thread">
                     <ThreadIcon class="o_ThreadViewTopbar_threadIcon mr-2 align-self-center" threadLocalId="threadViewTopBar.thread.localId"/>
                 </t>
-                <div class="o_ThreadViewTopbar_title d-flex flex-grow-1 align-self-center align-items-baseline">
+                <div class="o_ThreadViewTopbar_title d-flex flex-grow-1 align-self-center align-items-center">
```

Comment:
I think I'm ok with it to fix the flicker, but it looks less good when the description is no longer base aligned with the rest of the title.

So if there is another easy solution it would be better, but otherwise we don't have time to waste on that now.

Reply:
There is not really any simple alternative I can think of no. I would have to toy around with added containers.